test/e2e/storage/generic_persistent_volume-disruptive.go

```diff
@@ -106,9 +107,19 @@ func createPodPVCFromSC(f *framework.Framework, c clientset.Interface, ns string
 	framework.ExpectEqual(len(pvs), 1)
 
 	ginkgo.By("Creating a pod with dynamically provisioned volume")
-	pod, err := e2epod.CreateSecPod(c, ns, pvcClaims, nil,
-		false, "", false, false, e2epv.SELinuxLabel,
-		nil, framework.PodStartTimeout)
+	podConfig := e2epod.PodConfig{
+		Ns:                  ns,
+		Pvclaims:            pvcClaims,
+		InlineVolumeSources: nil,
```

Comment:
To make it easier to read, I think it would be good to omit the "default" values (nil, false, etc) from all the test cases that use this

Reply:
Okay. I will do that